widget-store/migrations/001_initial_schema.js

```diff
@@ -13,6 +13,7 @@ exports.up = async function(knex) {
     table.datetime('last_update_time').notNullable().defaultTo(knex.fn.now());
     table.integer('product_id').notNullable();
     table.foreign('product_id').references('products.product_id');
+    table.integer('progress_remaining').notNullable().defaultTo(10);
```

Comment:
Mention this new functionality somewhere, maybe both in the README and in the code.

Reply:
I updated the README with a description of what it does, and what happens on crash and after recovery. Does the update make sense?